src/languages/en.ts

```diff
@@ -1990,38 +2004,52 @@ export default {
-        receiptRequired: 'dummy.violations.receiptRequired',
-        rter: 'dummy.violations.rter',
-        smartscanFailed: 'dummy.violations.smartscanFailed',
-        someTagLevelsRequired: 'dummy.violations.someTagLevelsRequired',
-        tagOutOfPolicy: 'dummy.violations.tagOutOfPolicy',
-        taxAmountChanged: 'dummy.violations.taxAmountChanged',
-        taxOutOfPolicy: 'dummy.violations.taxOutOfPolicy',
-        taxRateChanged: 'dummy.violations.taxRateChanged',
-        taxRequired: 'dummy.violations.taxRequired',
+        allTagLevelsRequired: 'All tags required',
+        autoReportedRejectedExpense: ({rejectReason, rejectedBy}: ViolationsAutoReportedRejectedExpenseParams) => `${rejectedBy} rejected this expense with the comment "${rejectReason}"`,
+        billableExpense: 'Billable no longer valid',
+        cashExpenseWithNoReceipt: ({amount}: ViolationsCashExpenseWithNoReceiptParams) => `Receipt required over ${amount}`,
+        categoryOutOfPolicy: 'Category no longer valid',
+        conversionSurcharge: ({surcharge}: ViolationsConversionSurchargeParams) => `Applied ${surcharge}% conversion surcharge`,
+        customUnitOutOfPolicy: 'Unit no longer valid',
+        duplicatedTransaction: 'Potential duplicate',
+        fieldRequired: 'Report fields are required',
+        futureDate: 'Future date not allowed',
+        invoiceMarkup: ({invoiceMarkup}: ViolationsInvoiceMarkupParams) => `Marked up by ${invoiceMarkup}%`,
+        maxAge: ({maxAge}: ViolationsMaxAgeParams) => `Date older than ${maxAge} days`,
+        missingCategory: 'Missing category',
+        missingComment: 'Missing comment',
+        missingTag: ({tagName}: ViolationsMissingTagParams) => `Missing ${tagName ?? 'tag'}`,
+        modifiedAmount: 'Amount greater than scanned receipt',
+        modifiedDate: 'Date differs from scanned receipt',
+        nonExpensiworksExpense: 'Non-Expensiworks expense',
+        overAutoApprovalLimit: ({formattedLimitAmount}: ViolationsOverAutoApprovalLimitParams) => `Expense exceeds auto approval limit of ${formattedLimitAmount}`,
+        overCategoryLimit: ({categoryLimit}: ViolationsOverCategoryLimitParams) => `Amount over ${categoryLimit}/person category limit`,
+        overLimit: ({amount}: ViolationsOverLimitParams) => `Amount over ${amount}/person limit`,
+        overLimitAttendee: ({amount}: ViolationsOverLimitParams) => `Amount over ${amount}/person limit`,
+        perDayLimit: ({limit}: ViolationsPerDayLimitParams) => `Amount over daily ${limit}/person category limit`,
+        receiptNotSmartScanned: 'Receipt not verified. Please confirm accuracy.',
+        receiptRequired: ({amount, category}: ViolationsReceiptRequiredParams) => `Receipt required over ${amount} ${category ? ' category limit' : ''}`,
+        rter: ({brokenBankConnection, email, isAdmin, isTransactionOlderThan7Days, member}: ViolationsRterParams) => {
+            if (brokenBankConnection && isAdmin) {
```

Comment:
I made a mistake in what I sent you, and the message below is for when `$isAdmin` is false. With this fix, the copy below will be for when `brokenBankConnection && !isAdmin` which is right cause the submitter (not the approver/admin) is the one that needs to fix the bank connection, and the copy under `if (brokenBankConnection) {` is also right because in that case `$isAdmin` is true, and the message reflects that the employee, which is the submitter, needs to fix the bank connection
```suggestion
            if (brokenBankConnection && !isAdmin) {
```

Reply:
Ah then we can group the cases:

```ts
       if (brokenBankConnection) {
                return isAdmin
                    ? `Can't auto-match receipt due to broken bank connection which ${email} needs to fix`
                    : "Can't auto-match receipt due to broken bank connection which you need to fix";
        }
 ```

Reply:
And we can do the same below...

```
if (!isTransactionOlderThan7Days) {
                return isAdmin ? `Ask ${member} to mark as a cash or wait 7 days and try again` : 'Awaiting merge with card transaction.';
            }
```

Reply:
And in spanish:

```
rter: ({brokenBankConnection, isAdmin, email, isTransactionOlderThan7Days, member}: ViolationsRterParams) => {
            if (brokenBankConnection) {
                return isAdmin
                    ? `No se puede adjuntar recibo debido a una conexión con su banco que ${email} necesita arreglar`
                    : `No se puede adjuntar recibo debido a una conexión con su banco que necesita arreglar`;
            }
            if (!isTransactionOlderThan7Days) {
                return isAdmin
                    ? `Pídele a ${member} que marque la transacción como efectivo o espere 7 días e intente de nuevo`
                    : `Esperando adjuntar automáticamente con transacción de tarjeta de crédito`;
            }
            return ``;
        },
        ```